tests/test_features/test_pcoords.py

```diff
@@ -131,3 +157,65 @@ def test_integrated_pcoords(self):
         visualizer.fit_transform(X, y)
         visualizer.poof()
         self.assert_images_similar(visualizer)
+
+    @staticmethod
```

Comment:
What does making this a static method do for the test case?

Reply:
PEP8-compliance.  That is about it.